src/ui/components/Library/Team/View/NewTestRuns/TestRunsPage.tsx

```diff
@@ -98,16 +98,18 @@ function TestRunsContent() {
         <Panel minSize={20} order={1}>
           <div className="flex h-full w-full flex-col gap-4 overflow-hidden rounded-xl bg-bodyBgcolor p-2">
             <div className="flex flex-col gap-2">
-              <div className="grid w-full grid-cols-3 gap-2 bg-bodyBgcolor">
+              <div className="grid w-full grid-cols-3 gap-2 overflow-hidden bg-bodyBgcolor">
                 <div
```

Comment:
I thought this Dropdown was already a shared component (it probably should be but doesn't have to happen in this PR, I think). Since it isn't, lets confirm that the tests view doesn't also have this problem.

Reply:
Confirmed it doesn't have the same problem. Those dropdowns don't adjust at all and the row just overflows its container.

https://github.com/replayio/devtools/assets/15959269/0c509175-27f0-49a4-ac4d-e5c683f3efbd